lib/runcible/models/ostree_importer.rb

```diff
@@ -4,7 +4,7 @@ class OstreeImporter < Importer
       ID = 'ostree_web_importer'.freeze
       REPO_TYPE = 'OSTREE'.freeze
 
-      attr_accessor 'branches'
+      attr_accessor 'branches', 'depth'
```

Comment:
@jlsherrill found the answer 
```
> Runcible::Models::OstreeImporter.new(:depth => 10).config
=> {"depth"=>10}
> Runcible::Models::OstreeImporter.new(:feed => "http://ostree-house.org").config
=> {"feed"=>"http://ostree-house.org"}

> gg=Runcible::Models::OstreeImporter.new()
=> #<Runcible::Models::OstreeImporter:0x00000001403b88>
> gg.depth = 1
=> 1
gg.config
=> {"depth"=>1}
> gg.depth=nil
=> nil
> gg.config
=> {"depth"=>nil}
```


In short unless you forcibly set depth to nil, its going to work ok. Pulp will not get set the depth value if you don't provide the importer one. So imo no defaults are needed here.